pr-data.csv

```diff
@@ -809,10 +809,10 @@ https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-hdfs-project/hadoop-hdfs-nfs,org.apache.hadoop.hdfs.nfs.nfs3.TestWrites.testOverlappingWrites,OD-Vic,,,
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-hdfs-project/hadoop-hdfs-nfs,org.apache.hadoop.hdfs.nfs.nfs3.TestWrites.testWriteStableHow,OD-Vic,,,
 https://github.com/apache/hadoop,cc2babc1f75c93bf89a8f10da525f944c15d02ea,hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient,org.apache.hadoop.hdfs.TestNNBench.testNNBenchCreateReadAndDelete,ID,,,
-https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestClientCredentialTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,,,
-https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestRefreshTokenTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,,,
+https://github.com/apache/hadoop,14cd969b6ea1898e9db6eeb9ea5292ec4558a706,hadoop-hdfs-project/hadoop-hdfs-client,org.apache.hadoop.hdfs.web.oauth2.TestClientCredentialTimeBasedTokenRefresher.refreshUrlIsCorrect,ID,DeveloperFixed,,
```

Comment:
In which commit did the developer fix the test? You can have a look at https://github.com/TestingResearchIllinois/idoft/pull/635

Reply:
Oh, these tests are not flaky now. I've checked possible status of tests in idoft and suppose that these cases are suitable for `DeveloperFixed`. Should I change them to anther status?

Reply:
Yes, they're `DeveloperFixed` if they're still there but not flaky now. I'm saying it's better to find which commit did the developer fix the test